apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -604,6 +603,27 @@ const allFieldsHidden = (fieldVisibilities: AichatState['fieldVisibilities']) =>
     key => fieldVisibilities[key] === Visibility.HIDDEN
   );
 
+const updateMessagesOnError = (
+  newMessage: ChatCompletionMessage,
+  dispatch: ThunkDispatch<unknown, unknown, AnyAction>
+) => {
+  const assistantChatMessage: ChatCompletionMessage = {
+    id: getNewMessageId(),
+    role: Role.ASSISTANT,
+    status: Status.ERROR,
```

Comment:
I think of the "error response to your submission to model" as different than the "error occurred elsewhere and we're using the chat window to notify you of the error occurring". Looking at the [rough draft shared UI components for Code.org chatbots](https://docs.google.com/document/d/1kUWgyT_FbjCvXN2Lc28cMF6NWNrfv62O5SlVdjEIwFQ/edit), it does look like the preferred color for this situation would be red instead of yellow, so I updated to that. Thoughts on this?

![image](https://github.com/code-dot-org/code-dot-org/assets/25372625/2a64d05a-50cb-43ad-a24c-b0d957ab77b5)

Reply:
>I think of the "error response to your submission to model" as different than the "error occurred elsewhere and we're using the chat window to notify you of the error occurring". 

Yeah - that makes sense. Thanks!

And agree that it would be good to use the preferred color outlined in doc - good call!

Reply:
I now see that Emily [left a comment](https://docs.google.com/document/d/1kUWgyT_FbjCvXN2Lc28cMF6NWNrfv62O5SlVdjEIwFQ/edit?disco=AAABNnOTo9I) about this questioning the color. So maybe leave for now and ask Sam when she gets back or when conversation is resolved?

Reply:
It's already non-standard colored (ie, yellow), which I think is out of sync with Emily's observation anyway, so I'd be inclined to roll with what's currently laid out in the doc (ie, red)...happy to update once Mark weighs in though!